[SyntaxError]: Can not use 'await' as identifier inside an async function. (3,13)
3|    async ({await}) => {
 |           ^
[SyntaxError]: Can not use 'await' as identifier inside an async function. (3,13)
3|    async ({await}) => {
 |           ^
